package.json

```diff
@@ -200,6 +201,7 @@
   },
   "devDependencies": {
     "@actions/core": "1.5.0",
+    "@aws-sdk/client-s3": "3.35.0",
```

Comment:
Why is this needed?

Reply:
It's a peer dependency of the aws api mock packe I used for my tests... and i don't now which it was lint or test complained that the peer dependency was not found... i try to reproduce it.

Reply:
```
$ tsc --noEmit
node_modules/aws-sdk-client-mock/dist/types/libStorage.d.ts:8:95 - error TS2307: Cannot find module '@aws-sdk/client-s3' or its corresponding type declarations.

8 export declare const mockLibStorageUpload: (s3Mock?: import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes> | undefined) => import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes>;
                                                                                                ~~~~~~~~~~~~~~~~~~~~

node_modules/aws-sdk-client-mock/dist/types/libStorage.d.ts:8:143 - error TS2307: Cannot find module '@aws-sdk/client-s3' or its corresponding type declarations.

8 export declare const mockLibStorageUpload: (s3Mock?: import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes> | undefined) => import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes>;
                                                                                                                                                ~~~~~~~~~~~~~~~~~~~~

node_modules/aws-sdk-client-mock/dist/types/libStorage.d.ts:8:242 - error TS2307: Cannot find module '@aws-sdk/client-s3' or its corresponding type declarations.

8 export declare const mockLibStorageUpload: (s3Mock?: import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes> | undefined) => import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes>;
                                                                                                                                                                                                                                                   ~~~~~~~~~~~~~~~~~~~~

node_modules/aws-sdk-client-mock/dist/types/libStorage.d.ts:8:290 - error TS2307: Cannot find module '@aws-sdk/client-s3' or its corresponding type declarations.

8 export declare const mockLibStorageUpload: (s3Mock?: import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes> | undefined) => import("./awsClientStub").AwsStub<import("@aws-sdk/client-s3").ServiceInputTypes, import("@aws-sdk/client-s3").ServiceOutputTypes>;
```

Thats the error  I get without this dev dependency...

Reply:
So we can remove it? Or not? @rarkins you added it again, after it was removed?

Reply:
![image](https://user-images.githubusercontent.com/16692560/141656185-1184abee-996f-405b-9694-9bd55dda1154.png)
=> So I get this error after removing the dependency... now typecheck is failing :-/ sorry - I am a little bit lost with this

Reply:
that was the reason I added the dependency